funnel/views/helpers.py

```diff
@@ -42,6 +42,9 @@
 
 nocache_expires = utc.localize(datetime(1990, 1, 1))
 
+# Six avatar colours defined in _variable.scss
+no_avatar_colours = 6
```

Comment:
In the code above I kept reading it as "no" instead of "number". This variable should be named "avatar_color_count" (using American English for variable names).